src/charts/vertical_bar.coffee

```diff
@@ -516,10 +516,15 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
       .tickSize(@get 'graphicWidth')
       .tickFormat(@get 'formatValueAxis')
 
+    gYAxis = @get('yAxis')
+
+    # find the correct size of graphicLeft in order to fit the Labels perfectly
+    @set 'graphicLeft', @maxLabelLength(gYAxis.selectAll('text')) + @get 'labelPadding'
```

Comment:
This same code is in multiple charts, consider moving it to a mixin if possible.